geoevent-components/solutions-geoevent/adapters/regexText-adapter/src/main/java/com/esri/geoevent/solutions/adapter/regexText/RegexTextInboundAdapter.java

```diff
@@ -25,91 +25,102 @@
 import java.util.regex.Pattern;
 
 import org.apache.commons.lang3.StringUtils;
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
 
-import com.esri.ges.adapter.text.TextInboundAdapter;
 import com.esri.ges.adapter.AdapterDefinition;
+import com.esri.ges.adapter.text.TextInboundAdapter;
 import com.esri.ges.core.component.ComponentException;
-
+import com.esri.ges.framework.i18n.BundleLogger;
+import com.esri.ges.framework.i18n.BundleLoggerFactory;
 
 public class RegexTextInboundAdapter extends TextInboundAdapter
 {
-	// This class enables an attribute separator to be defined as a regular expression (which the base class does not handle).
-	// It swaps all matches of this regular expression in the incoming stream with a single-character delimiter (which the base class will handle)
-	// It then passes the altered stream to the base class to handle as normal
-	static final private Log log = LogFactory.getLog(RegexTextInboundAdapter.class);
-	private String attributeSeparator;
-	protected static String ATTRIBUTE_SEPARATOR_TO_BE_REPLACED_PROPERTY_NAME = "AttributeSeparatorToBeReplaced";
-	private String separatorToBeReplaced = "";
-	private static Charset charset = Charset.forName("UTF-8");
-	
-	public RegexTextInboundAdapter(AdapterDefinition adapterDefinition)
-			throws ComponentException
-	{
-		super(adapterDefinition);
-	}
+  // This class enables an attribute separator to be defined as a regular
+  // expression (which the base class does not handle).
+  // It swaps all matches of this regular expression in the incoming stream with a
+  // single-character delimiter (which the base class will handle)
+  // It then passes the altered stream to the base class to handle as normal
+  static final private BundleLogger LOGGER                                           = BundleLoggerFactory.getLogger(RegexTextInboundAdapter.class);
+  private String                    attributeSeparator;
+  protected static String           ATTRIBUTE_SEPARATOR_TO_BE_REPLACED_PROPERTY_NAME = "AttributeSeparatorToBeReplaced";
+  private String                    separatorToBeReplaced                            = "";
+  private static Charset            charset                                          = Charset.forName("UTF-8");
+
+  public RegexTextInboundAdapter(AdapterDefinition adapterDefinition) throws ComponentException
+  {
+    super(adapterDefinition);
+  }
+
+  @Override
+  public void afterPropertiesSet()
+  {
+    super.afterPropertiesSet();
+    // grab references to two properties in the definition that will be used to
+    // adapt the incoming bytes
```

Comment:
Please increase your allowed line length for comments so this line is on one line